client/me/purchases/remove-purchase/remove-domain-dialog/index.jsx

```diff
@@ -52,12 +53,15 @@ class RemoveDomainDialog extends Component {
 						components: { strong: <strong /> },
 					}
 				) }
-				{ hasTitanWithUs &&
-					' ' +
-						translate(
+				{ hasTitanWithUs && ' ' + isEnabled( 'titan/phase-2' )
+					? translate(
 							'You also have an active Titan Mail subscription for this domain, and your emails will stop ' +
 								'working if you delete your domain.'
-						) }
+					  )
+					: translate(
+							'You also have an active Email subscription for this domain, and your emails will stop ' +
+								'working if you delete your domain.'
+					  ) }
```

Comment:
I would suggest using a variable similar to `googleMailService` in strings, for the following reasons:

1. It will reduce the work of translators if we ever decide to rename that product
2. It will make it easier to spot places where we actually use that product name

Reply:
I've refactored this to use a helper function, with one or two small exceptions that use sentences.